dashboards/grafana-dashboard-acs-fleet-manager-slos.configmap.yaml

```
@@ -962,7 +962,7 @@ data:
               },
               "editorMode": "code",
               "exemplar": false,
-              "expr": "sum(acs_fleet_manager_worker_central_duration_bucket{namespace=\"$namespace\",jobType=\"central_create\"}) by (le)",
+              "expr": "sum(increase(acs_fleet_manager_worker_central_duration_bucket{namespace=\"$namespace\",jobType=\"central_create\"}[$__range])) by (le)",
```

Why use the `increase` function? I thought that `rate` is usually used in this case. 

```suggestion
              "expr": "sum(rate(acs_fleet_manager_worker_central_duration_bucket{namespace=\"$namespace\",jobType=\"central_create\"}[$__range])) by (le)",
```

---

`rate` is not suitable here because we need all numeric value combine and not an average from them. So the point of this chart is to provide how many centrals were provisioned under certain period